src/frontend-admin/src/components/App/App.tsx

```diff
@@ -13,39 +13,45 @@ export const App: React.FC = () => {
       <Router>
         <div className={styles.navContainer}>
           <nav className={styles.navBar}>
-          
             <h1>CQL</h1>
-            
             <div className={styles.linkContainer}>
               <NavLink className={styles.navTab} 
-                activeClassName={styles.active} 
-                to="/dashboard">
-                <label>Dashboard</label>
+                activeClassName={styles.active}
+                to="/departments">
+                <div className={styles.navTabRectangle} />
+                <label>Departments</label>
               </NavLink>
 
               <NavLink className={styles.navTab} 
                 activeClassName={styles.active} 
-                to="/employees">
-                  <label>Employees</label>
+                to="/hardware">
+                <div className={styles.navTabRectangle} />
+                <label>Hardware</label>
               </NavLink>
 
               <NavLink className={styles.navTab} 
                 activeClassName={styles.active} 
                 to="/programs">
+                <div className={styles.navTabRectangle} />
                 <label>Programs</label>
               </NavLink>
 
               <NavLink className={styles.navTab} 
                 activeClassName={styles.active} 
-                to="/hardware">
-                <label>Hardware</label>
+                to="/employees">
+                  <div className={styles.navTabRectangle} />
+                  <label>Employees</label>
               </NavLink>
 
               <NavLink className={styles.navTab} 
-                activeClassName={styles.active}
-                to="/departments">
-                <label>Departments</label>
+                activeClassName={styles.active} 
+                to="/dashboard">
+                <div className={styles.navTabRectangle} />
+                <label>Dashboard</label>
               </NavLink>
+
+              <div className={styles.navEllipse} />
+              <div className={styles.navRectangle} />
```

Comment:
You _probably_ don't need to do this but the combo of these two divs almost looks good enough to be a component to me. I don't think we'll reuse it elsewhere in the app but I suppose the wires are not complete yet. Just something to keep in mind for now.